_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,353 @@
+  interplay between privacy requirements, accessibility, storage, and
+  fees charged.  Delgado is working on the watchtower implementation
+  [The Eye of Satoshi][teos] at Talaia Labs, which is aiming to be
+  compliant with the proposed BOLT13 (see [Newsletter #75][news75
+  watchtower bolt]).  Delgado also highlighted how watchtowers are
+  becoming increasingly critical in multiple settings such as Bitcoin
+  [vault][topic vaults] designs, statechains, and atomic swaps in addition to LN.
+  ([transcript][watchtowers ts], [video][watchtowers vid],
+  [slides][watchtowers slides])
+
+- **CoinSwap:** Aviv Milner presented on CoinSwap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  CoinSwap proposal (see [Newsletter #100][news100 coinswap]) provides
+  covertness in a manner that other privacy schemes such as
+  [coinjoin][topic coinjoin] fail to do. Milner also went through the
+  motivation for routing CoinSwaps, namely to address the concern of
+  unwittingly entering into a CoinSwap with an adversary such a chain
+  surveillance company.  ([transcript][coinswap ts], [video][coinswap
+  vid])
+
+- **Schnorr signatures and multisignatures:** [BIP340][] co-authors Tim
+  Ruffing, Pieter Wuille, and Jonas Nick held a discussion at London
+  Bitcoin Devs on the specification of [schnorr signatures][topic
+  schnorr signatures].  This covered earlier ideas for implementing
+  schnorr signatures in Bitcoin and what the co-authors thought the
+  community should be concerned about when considering a possible future
+  soft fork deployment. Wuille noted that he is most concerned about how
+  schnorr is adopted and what is built using it.  The following day, Tim
+  Ruffing presented on the challenges of multisignature and threshold
+  signature schemes using schnorr signatures.  He's been working with
+  collaborators on a speculative [MuSig][topic musig] signature scheme
+  that only requires two rounds of interaction rather than three,
+  without relying on zero knowledge proofs, which would greatly simplify
+  some threshold signing workflows.  ([Meeting transcript][socratic ts],
+  [meeting video][socratic vid], [presentation transcript][ruffing ts],
+  [presentation video][ruffing vid], [presentation slides][ruffing
+  slides])
+
+- **Sydney meetup discussion:** A number of Bitcoin and LN developers
+  joined this Sydney meetup to discuss various topics. Ruben Somsen gave
+  a short presentation on Succinct Atomic Swaps (SAS) (see [Newsletter
+  #98][news98 sas]) and how it compares to Chris Belcher’s CoinSwap
+  proposal.  Another topic was Bitcoin Core's reliance on DNS seeds to
+  find initial peers, whether that's an attack vector against newly
+  started nodes, and how work by Matt Corallo and Antoine Riard on
+  allowing [alternative transports][Bitcoin Core #18988] could help
+  mitigate any risk.  Finally, Lloyd Fournier discussed how
```

Comment:
"any risk" seems off

Reply:
Mitigate means lessen, so it is certainly not saying all risk is eradicated. But perhaps s/any risk/this risk would be an improvement.